.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will use tox to run the unit tests on every push and pull_request event
+
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox wheel
+    - name: Run the tests with tox
+      run: |
+        make test
+        python setup.py sdist bdist_wheel install
```

Comment:
@tumb1er - I took this line from here: https://github.com/just-work/django-celery-rpc/blob/8aed013b89f571880835fe0e043f012779db126a/.github/workflows/build.yml#L26 , but do not really understand the requirement of this `install` step. Does this in some way ensure some build sanity or install testing?

Reply:
It's explicit build/install smoke test.